packages/ui-components/src/chat/message.tsx

```diff
@@ -177,7 +197,11 @@ function MessageComponent(props: MessageProps): JSX.Element {
             {!editMode && (
                 <MessageBody>
                     {localMessage.message}
-                    {localMessage.edited && <EditedText> (edited)</EditedText>}
+                    {localMessage.updated_at && (
```

Comment:
If you set `updated_at` at creation time (equal to `created_at`) then you don't need any null checks, and here you just check if `created_at === updated_at`

Reply:
But we only keep up to the minutes at the moment for the timestamp, for this condition to be valid we would need to keep seconds at least as well, this seemed like a better check?